containers/dev/docker-compose.yml

```diff
@@ -11,26 +11,41 @@ services:
     ports:
       - "9200:9200"
       - "9300:9300"
-#  kibana-dev:
-#    image: kibana:5.1
-#    ports:
-#      - "5601:5601"
-#    environment:
-#      - ELASTICSEARCH_URL=http://elasticsearch-dev:9200
-#    links:
-#      - elasticsearch-dev
-#  zookeeper:
-#     image: wurstmeister/zookeeper
-#     ports:
-#       - "2181:2181"
-#  kafka:
-#    image: wurstmeister/kafka
-#    build: .
-#    ports:
-#      - "9092:9092"
-#    environment:
-#      KAFKA_ADVERTISED_HOST_NAME: 127.0.0.1
-#      KAFKA_CREATE_TOPICS: "ctia-events:1:1"
-#      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+  zookeeper:
+    image: confluentinc/cp-zookeeper:latest
+    hostname: "zookeeper"
+    environment:
+      ZOOKEEPER_SERVER_ID: 1
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+      ZOOKEEPER_INIT_LIMIT: 5
+      ZOOKEEPER_SYNC_LIMIT: 2
+      ZOOKEEPER_SERVERS: localhost:2181:2181
+    ports:
+      - "2181:2181"
+
+  kafka:
```

Comment:
The current configuration does not seem to be able to work with a `scale` via docker-compose. I think we should create an issue to enhance that to be able to find potential bug only present with multiple brokers.